int8 enable
int8 run
int8 gui_connection_requierd=1
float64 max_speed=3.0
time time_frame
